### ISSUE TITLE
Accessibility issue: Electron system tray context menu on Windows is not keyboard navigable

### ISSUE THREAD

**Issue by josimi** [electron/electron#11587]
* Electron version: 1.7.10
* Operating system: Windows 10

### Expected behavior

It is possible to access system tray items and their respective context menus on Windows using the keyboard only. This functionality is important for accessibility reasons.

The system tray is selected using the key combination **WINDOWS+B** and then opened using the **ENTER** key. It is then possible to navigate to a specific icon using the arrow keys and select it with **ENTER** to open the icon's context menu. It _should_ be possible to then navigate through this menu using the arrow keys and select a specific item with **ENTER**.

### Actual behavior

Arrow key navigation and ENTER selection is not available for Electron apps' system tray icon context menus. Pressing the arrow keys while the system tray icon's context menu is open, after being opened using the keyboard steps above, produces no result.

### How to reproduce

1. **WINDOWS+B** and then **ENTER** to open the list of icons
2. Arrow keys and to select the Electron app's icon
3. Press **SHIFT+F-10** to open the icon's context menu
4. Observe that arrow keys are not useable within this icon's menu, but are for other icons.

### Technical details

See **notify_icon.cc**'s function `NotifyIcon::PopUpContextMenu` which makes a call to the `->RunMenuAt`. I am new to the Electron codebase, but as far as I can tell, this is `RunMenuAt` call is defined in Chromium, meaning we can't change it directly, but will need to find another way to show context menus that is keyboard accessible.

**Comment by josimi** [comment]
Looks like the missing key press handling functionality should be in Chromium's `/ui/views/controls/menu/menu_controller.cc`.

**Comment by brentgti** [comment]
Did regression occur with this? I created an application with a tray menu, and I can't navigate it with the arrow keys on my keyboard. My code is here: https://github.com/brentgti/electron-Teams-links.

**Comment by brentgti** [comment]
@josimi, is this issue actually fixed, or maybe there's a regression issue here? My tray menu doesn't allow for navigation via keyboard. See https://github.com/brentgti/electron-Teams-links.

**Comment by danikane** [comment]
I second that. Can't navigate with the keyboard.

- Electron v7.0.1
- electron-builder 21.2.0

**Comment by danikane** [comment]
Update:
Upgraded to Electron v8.1.1 and electron-builder 22.4.1, same behavior.
Thanks!

**Comment by brentgti** [comment]
I still can't navigate the menu in my application with the arrow keys. I'm using electron 9.0.0. Others have referred to an electron-builder version, but I don't know about that. I'm using electron-packager to create my .exe.

**Comment by danikane** [comment]
The fix was merged just 3 days ago, man...
Wait for a new release.

**Comment by brentgti** [comment]
Sorry, I forgot how these things work 😳. Thanks for the reply.

**Comment by danikane** [comment]
@brentgti https://github.com/electron/electron/releases/tag/v9.0.2

**Comment by ckerr** [comment]
Reopening this issue since the previous fix had unintended side-effects.
